VATSpy.dat

```diff
@@ -18270,9 +18253,6 @@ VVVV|Hanoi||VVVV
 VVTS|Ho-Chi-Minh||VVTS
 VYYF|Rangoon|VYYY|VYYF
 WAAF|Ujung Pandang||WAAF
-WAAF-B|Ujung Pandang (Bali)|WAAF_B|WAAF_B

```

Comment:
WAAF-B, WAAF-W and WAAF-E exist as FIR-Boundaries

https://github.com/vatsimnetwork/vatspy-data-project/blob/f840d74eee3032cc77da5116dbd4563aad6009e6/FIRBoundaries.dat#L20443

https://github.com/vatsimnetwork/vatspy-data-project/blob/f840d74eee3032cc77da5116dbd4563aad6009e6/FIRBoundaries.dat#L20451

https://github.com/vatsimnetwork/vatspy-data-project/blob/f840d74eee3032cc77da5116dbd4563aad6009e6/FIRBoundaries.dat#L20507